src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -222,13 +228,30 @@ export default function createAnimatedComponent(
       this._detachStyles();
       this._detachInlineProps();
       this._sharedElementTransition?.unregisterTransition(this._viewTag);
+
+      if (IS_WEB) {
+        handleWebAnimation(
+          this.props,
+          this._component as HTMLElement,
+          LayoutAnimationType.EXITING
+        );
+      }
     }
 
     componentDidMount() {
       this._attachNativeEvents();
       this._JSPropUpdater.addOnJSPropsChangeListener(this);
       this._attachAnimatedStyles();
       this._attachInlineProps();
+
+      if (IS_WEB) {
+        insertWebAnimations();
```

Comment:
What does this function do apart from injecting CSS styles? What about `configureWebLayoutAnimations`?

Reply:
Done in [1b57aa1](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/1b57aa1a29d6a706a96ee84f23fc8a426d27ec22)